xml/schema/Contact/ACLContactCache.xml

```diff
@@ -7,18 +7,6 @@
   <comment>Join table cache for contacts that a user has permission on.</comment>
   <add>3.1</add>
   <field>
-    <name>id</name>
-    <title>ACL Contact Cache ID</title>
-    <type>int unsigned</type>
-    <required>true</required>
-    <comment>primary key</comment>
-    <add>3.1</add>
-  </field>
-  <primaryKey>
-    <name>id</name>
-    <autoincrement>true</autoincrement>
-  </primaryKey>
-  <field>
```

Comment:
Instead of deleting these from the xml completely, use the `<drop>` tag. Search for `<drop>` in schema xml files for examples.